### DISCUSSION TITLE
Εγκατάσταση το Arch Linux με Virtualbox;

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1616] [Announcements]
Μπορούμε να κάνουμε εγκατάσταση το Arch Linux με Virtualbox; Τα βήματα και η διαδικασία είναι ίδια μεταξύ usb ή VirtualBox.

Και στις 2 περιπτώσεις πρέπει να κάνουμε: 
1. Format partitions
2. Create directories και mount τα partitions
3. Install basic packages όπως linux, linux-firmware, vim and base
4. Δημιουργία filesystem table
5. Δημιουργία hostname, timezone και system locale
6. Δημιουργία root password
7. Εγκατάσταση grub, efibootmgr και networkmanager
8. Δημιουργία image και config file για το grub
9. Enable network
10. Δημιουργία χρήστη
11. Αλλαγή του wheel με vim για sudo access από τον χρήστη
12. Unmount τα partitions
13. Restart system

Κάνω αυτό το discussion εδώ, για να το δουν όσοι πήγαν με VirtualBox, έτσι ώστε να μην τρέχουν τελευταία στιγμή να ξανά κάνουν την εγκατάσταση σε USB ή Dual Boot, αν δεν επιτρέπετε το VirtualBox.

**Comment by epidrome** [comment]
κοπι-πεηστ από τις οδηγίες (=κανονικά θα έπρεπε να αγνοήσουμε-σβήσουμε αυτήν την ερώτηση):
https://courses-ionio.github.io/projects/dokey/

> π.χ., την 1η εβδομάδα θα πρέπει να εντοπίσετε τα χαρακτηριστικά που έχει το υλικό σας (δίσκος, μνήμη, επεξεργαστής, δίκτυο, γραφικά, κτλ)

> Επομένως, δεν γίνονται δεκτές εκδοχές του Arch, όπως το δημοφιλές Manjaro που βασίζονται σε αυτόματη εγκατάσταση, ούτε και εγκατάσταση σε εικονικές μηχανές, παρά μόνο εγκατάσταση σε αληθινό υλικό.

μερικά ακόμη σχόλια, για να γίνουν κατανοητά τα οφέλη από αυτήν την άσκηση:
https://unix.stackexchange.com/questions/140908/is-installing-and-configuring-arch-linux-in-virtualbox-going-to-be-the-same-as-i

πράγματι, η εγκατάσταση σε εικονική μηχανή είναι λίγο πιο εύκολη, και σίγουρα χρήσιμη για ζέσταμα, αλλά

δεν αντικαθιστά το ζητούμενο που είναι η εγκατάσταση σε αληθινό υλικό

ένα ζέσταμα σε εικονική μηχανή ποτέ δεν είναι λάθος, αλλά ούτε είναι απολύτως σωστό, αναφορικά με τις γνώσεις-δεξιότητες που προσφέρει το αληθινό υλικό

επομένως, αυτό που περίπου ρωτάς είναι: "θα πείραζε να μην μάθω όλες τις γνώσεις-δεξιότητες της εγκατάστασης";

αντίστοιχα λειτουργεί και η βαθμολόγηση αυτής της άσκησης, άρα η εικονική μηχανή είναι δεκτή, αλλά θα βαθμολογηθεί στην βάση

**Reply by mariosconsta** [reply]
Κατανοητό, θα ξανά κάνω εγκατάσταση σε USB!
